test/studies/bale/histogram/histo-atomics-forall-opt.ml-execopts

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env python
+
+# Run bale histo with 2 million updates per task. non-ugni configs have much
+# slower network atomics, so drop to 20,000 updates per task
+import os
+
+comm = os.getenv('CHPL_COMM')
+ugni = comm == 'ugni'
+
+N = 20000
+if ugni:
+  N = 2000000
+
+print('--N={0} --printStats --useBufferedAtomics=false # bale-hist-atomic'.format(N))
+print('--N={0} --printStats --useBufferedAtomics=true  # bale-hist-buff-atomic'.format(N))
```

Comment:
Why bother to run/graph the non-buffered vs. buffered cases here. I think we should just run with `--useBufferedAtomics=false` (to save runtime and to prevent the graphs from getting too busy)